qiime2/core/cite.py

```diff
@@ -26,6 +26,16 @@
 """
 
 
+def make_citations_tuple(citations):
```

Comment:
@colinvwood, any reason to not make this a private (`_make_citations_tuple`) function?

Reply:
I guess because it's used outside of the module it's defined in. But we mainly use underscores just to differentiate between the public api and everything else huh? I can change it in that case.

Reply:
Yeah, that's the typical way we use it.